### ISSUE TITLE
SFT with large loss {'loss': 388082722196684.8, 'learning_rate': 0.0, 'epoch': 0.02}

### ISSUE THREAD

**Issue by LeiShenVictoria** [jackaduma/Vicuna-LoRA-RLHF-PyTorch#11]
The SFT loss is very large and becomes inf then nan at last.
{'loss': 240023.775, 'learning_rate': 0.0002762320648783531, 'epoch': 0.47}
{'loss': inf, 'learning_rate': 0.0002743605739238927, 'epoch': 0.49}
{'loss': nan, 'learning_rate': 0.00027248908296943227, 'epoch': 0.51}
